src/components/views/settings/encryption/RecoveryPanel.tsx

```diff
@@ -21,14 +21,17 @@ import { SettingsSubheader } from "../SettingsSubheader";
  * - `loading`: We are checking the recovery key and the secrets.
  * - `missing_recovery_key`: The user has no recovery key.
  * - `secrets_not_cached`: The user has a backup but the secrets are not cached.
- *                         This shouldn't happen but we have seen cases where the secrets gossiping failed or shared partial secrets when verified with another device.
- * - `good`: The user has a backup and the secrets are cached.
+ *                         This can happen if we verified another device and secret-gossiping failed, or the other device itself lacked the secrets.
```

Comment:
It's worth noting that this indentation is lost if/when the tsdoc is rendered to HTML, and is inconsistent with our style elsewhere.

But, not a big problem.